quest-editor/quest-editor-webapp/src/main/webapp/quest-editor/editor.xhtml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://java.sun.com/jsf/html"
+      xmlns:f="http://java.sun.com/jsf/core"
+      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:p="http://primefaces.org/ui">
+
+<h:head>
+    <title>Hello World!</title>
+</h:head>
+<h:body>
+    <h:form id="form">
+
+        <p:growl id="msgs" showDetail="true" />
+
+        <h3 style="margin-top:0">Test Quest v0.1</h3>
+        <button style="margin: 2px;">Сохранить</button>
+        <p:tabView>
+            <p:tab  title="Квест">
```

Comment:
Could you see how we may extract code from tab to another sheets.
To many code for one page. Try to divide to multiple layers with one master page layer.